.travis.yml

```diff
@@ -38,3 +38,4 @@ install:
 script:
   - ./ci/run_stable_checks.sh
   - GECKODRIVER=$(pwd)/geckodriver ./ci/run_tests.sh
+# DISABLED - ./ci/build_examples.sh
```

Comment:
We already check all examples in CI, we don't need to build them in CI. Could you please remove this line?

Reply:
removed